frontend/src/scenes/sessions/filters/EditFiltersPanel.tsx

```diff
@@ -73,6 +77,79 @@ export function EditFiltersPanel({ onSubmit }: Props): JSX.Element | null {
             AND
         </span>
     )
+    const properties: PropertyFilter[] = []
+    const events: Entity[] = []
+    const actions: Entity[] = []
+
+    Object.entries(displayedFilters).forEach(([key, values]) => {
+        if (key === ACTION_TYPE) {
+            values.forEach((filter) => {
+                if (
+                    filter.item.label &&
+                    (typeof filter.item.value === 'string' || typeof filter.item.value === 'number')
+                ) {
+                    actions.push({
+                        id: filter.item.value,
+                        name: filter.item.label,
+                        type: 'actions',
+                        order: 0,
+                    })
+                }
+            })
+        }
+        if (key === EVENT_TYPE) {
+            values.forEach((filter) => {
+                if (
+                    filter.item.label &&
+                    (typeof filter.item.value === 'string' || typeof filter.item.value === 'number')
+                ) {
+                    events.push({
+                        id: filter.item.value,
+                        name: filter.item.label,
+                        type: 'events',
+                        order: 0,
+                    })
+                }
+            })
+        }
+        if (key === 'cohort') {
+            values.forEach((filter) => {
+                if (filter.item.value) {
+                    properties.push({
+                        key: 'id',
+                        value: filter.item.value,
+                        type: 'cohort',
+                        operator: 'exact',
+                    })
+                }
+            })
+        }
+        if (key === 'person') {
+            values.forEach((filter) => {
+                if (filter.item.value) {
+                    properties.push({
+                        key: filter.item.key,
+                        value: filter.item.value,
+                        type: 'person',
+                        operator: (filter.item as PersonPropertyFilter).operator,
+                    })
+                }
+            })
+        }
+    })
+
+    const params = {
```

Comment:
Since there's a lot of this kind of link utility code in here and other components, maybe we could set a more standardized/reusable way for this with a utility helper class or something similar in a future PR